x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -276,6 +276,14 @@ export interface SafeResolverRelatedEvents {
   nextEvent: string | null;
 }
 
+/**
+ * Response structure for the events route.
```

Comment:
When `nextEvent` is null it means there are no more events, right? Maybe should add that to comment.

Reply:
Yeah I'll add it 👍